src/components/Header/Header.jsx

```diff
@@ -116,7 +116,7 @@ const SettingsGroup = ({ children }) => {
   const [anchorEl, setAnchorEl] = useState(null);
   const handleClose = useCallback(() => {
     setAnchorEl(undefined);
-  }, [anchorEl]);
+  }, []);
```

Comment:
What does this change?

Reply:
Removes the dependency of anchorEl which was wrong.

But I think we need `setAnchorEl` as a dependency since we use it in the callback function.

Reply:
Sorry I don't have that much experience with hooks. It sounds like from reading the docs/your answer, that I need to pass through setAnchorEl to the dependency list in that case? So the original one must have been a typo. Am I correct?

Reply:
Once that's verified I can push the change

Reply:
Actually sorry I've misinterpretted the docs. The code looked funtionality correct before - as we're waiting for any changes to anchorEl done through the callback function which is passed to the onClose of the dropdown menu. I will undo these changes and put in a linter ignore for the time being until someone with a bit more experience can help

Reply:
`[setAnchorEl]` should be fine since it's the only dependency we require in the callback function itself

Reply:
Ah yep ok now I get what you mean. Apologies, I'm relatively new to hooks and am still learning them. Appreciate the help :), pushing fix for the linting error now

Reply:
It's all cool, hooks are a pretty new concept. I'm also still learning. :)